### ISSUE TITLE
Poor performance in IE11

### ISSUE THREAD

**Issue by crismethods** [gmaps-marker-clusterer/gmaps-marker-clusterer#57]
Hey guys!

We are using your library for thousands of markers at once, some clusters have 2k or more (zoom level 2), and we have experienced poor performances when using IE11, compared with Chrome and FF in IE it took up to 3 times for finishing loading the markers into the clusters. We are wondering if you could give us some advises in order to set maybe some parameters in order to achieve similar times between these three browsers.

Thanks in advance!

**Comment by XmlmXmlmX** [comment]
Hi @crismethods , do you have an example?

**Comment by crismethods** [comment]
Hey @XmlmXmlmX, yes for instance for the same case, as you can see in the images attached, the results are almost 5 times different comparing IE 11 against Chrome (**5058** ms VS **1267** ms). The code printing those time differences is:

```
    let init_date = Date.now();
    console.log("starting updateData");
    console.log(init_date);
    console.log(markers.length);
    this.addMarkersToCluster(markers);

    if (clusterFeatureAdded || clusterFeatureRemoved) {
      this.markerClusterer.repaint();
      // this.markerClusterer.resetViewport();
    }
    let end_date = Date.now();

    console.log("finishing updateData");
    console.log(end_date);
    console.log(end_date - init_date);
```

There are 15.001 objects.
![screen shot 2017-11-14 at 15 09 00](https://user-images.githubusercontent.com/31204546/32784294-b541d47e-c94e-11e7-902b-c6aa3cddf920.png)
![screen shot 2017-11-14 at 15 11 33](https://user-images.githubusercontent.com/31204546/32784295-b55a2484-c94e-11e7-9d84-5f49080b163d.png)

Thanks for your comments.

Cheers

**Comment by NomanBashir133** [comment]
Any Progress or workaround for this one.

**Comment by crismethods** [comment]
@NomanBashir133 I think this project is dead :( no commits since quite a while.

**Comment by Remo** [comment]
@crismethods I'm still keeping an eye on this repo, but as always with open source, priorities change. I currently don't have any use for this library and like most people, my spare time is limited. I will review pull requests if someone creates one, but I'm currently in no position to analyse this problem myself. I can obviously only speak for myself and not Sebastian.

**Comment by crismethods** [comment]
yes @Remo , I totally understand :) there are some PRs opened, one is for myself actually, nothing too big, but it was something we needed in our company. About the performance, we managed to deal with it in chunks and then it worked well, I wish I could also open PRs for improvements, but as you said, I am out of google maps at the moment.
